docs/getting-started/introduction.md

```diff
@@ -6,32 +6,14 @@ from pcweb.pages.docs import tutorial
 from pcweb.pages.docs import getting_started
 from pcweb.pages.docs import wrapping_react
 from pcweb.pages.docs.library import library
+from pcweb.pages.docs import vars
 ```
 
-# Introduction
+# Pure Python Web Apps
```

Comment:
I think "Introduction" may be good here actually.

https://nextjs.org/docs
https://svelte.dev/docs/introduction

Also, after the sidebar flattening change, maybe this page can just be called "Getting Started" instead